platforms/r3-corda-ent/charts/zone/values.yaml

```diff
@@ -0,0 +1,231 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# Default values for CENM Zone service.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+#############################################################
+#                   Basic Configuration                     #
+#############################################################
+# Provide the name of the deployment
+# Eg. nodeName: zone
+nodeName: 
+
+# Display more messages while executing Kubernetes deployment
+bashDebug: false
```

Comment:
This key has never been used. Please check all keys in values.yaml are used in templates. or delete them if not needed

Reply:
fixed